packages/sanes-chrome-extension/src/extension/background/messages/index.ts

```diff
@@ -81,11 +81,14 @@ export async function sendGetPersonaMessage(): Promise<GetPersonaResponse> {
   });
 }
 
+export type GetRequestResponse = ReadonlyArray<Request>;
+
 /**
  * String values will be used in the JSON message objects
  */
 export enum MessageToForegroundAction {
   TransactionsChanges = 'transaction_changed',
+  RequestChanges = 'request_changed',
```

Comment:
Ahh, sorry for my stupid naming above. It should be "transactions changed" for both the symbol and the string, to say: _here is a signal telling you that something happened in the past, feel free to react to that_. I'll update the existing code.